packages/devtools_app/lib/src/ui/analytics.dart

```diff
@@ -307,9 +307,12 @@ Future<bool> get isSurveyActionTaken async {
 
 /// Set DevTools property value 'surveyActionTaken' stored in the file
 /// '~\.devtools'.
+// TODO(terry): remove the query param logic for this request.
+// setSurveyActionTaken should only be called with the value of true, so
+// we can remove the extra complexity.
 void setSurveyActionTaken([bool value = true]) async {
   final resp = await HttpRequest.request(
-    // Format of request is e.g., api/setDevToolsEnabled?surveyActionTaken=true
+    // Format of request is e.g. api/setSurveyActionTaken?surveyActionTaken=true
```

Comment:
instead of a TODO can you just remove the parameter from the api?

Reply:
Good idea remove from this helper function always push true.

Reply:
We can do this, but there is still some unnecessary complexity in the server handler. I'll leave the TODO for cleanup purposes, but remove the param.